lib/routes/iheima/index.ts

```diff
@@ -0,0 +1,45 @@
+import { Route } from '@/types';
+
+import got from '@/utils/got';
+import { parseDate } from '@/utils/parse-date';
+
+export const route: Route = {
+    path: '/recommend',
+    categories: ['new-media'],
+    example: '/iheima/recommend',
+    url: 'https://www.iheima.com/',
+    name: 'i黑马网 - 推荐',
```

Comment:
Do not repeat namespace name https://github.com/DIYgod/RSSHub/blob/244a9b78f6fbe99eaa3efaafc6230c96fd28f330/lib/routes/iheima/namespace.ts#L4 in route name

Reply:
```typescript
    return {
        title: '推荐',
        link: baseUrl,
        item: items,
    };
```

Does the title here need to include the name of the site?